Bottom: 0042aa7c1748013a824d705362ecefb288487bc9
Top:    f5cd1a8791642c78c3b019205126d605508849b8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-14 13:26:30 +0100

Refresh of shiny-3d-eye-candy

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index be82e4e..854f413 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -2674,7 +2674,6 @@ ghid_parse_arguments (int *argc, char ***argv)
   gtk_gl_init(argc, argv);
 
   gport = &ghid_port;
-  fprintf (stderr, "gport set at %p\n", gport);
   gport->zoom = 300.0;
   pixel_slop = 300;
